examples/jsm/nodes/core/NodeBuilder.js

```diff
@@ -297,7 +297,7 @@ class NodeBuilder {
 
 	hasGeometryAttribute( name ) {
 
-		return this.geometry?.getAttribute( name ) !== undefined;
+		return this.geometry ? ( this.geometry.getAttribute( name ) !== undefined ) : undefined;
```

Comment:
This should be `this.geometry && this.geometry.getAttribute( name ) !== undefined`.

Reply:
> Meaning when `this.geometry` is undefined, the method used to return `undefined`.

I think `this.geometry?.getAttribute( name ) !== undefined` is `false` in this case, not `undefined`.

Reply:
You're right! I've actually misread the statement.